File: php_tasks/task5.php
<?php
    $a =readline("Enter a: ");
    $b =readline("Enter b: ");
    $c =readline("Enter c: ");
    if ($a > $c && $b > $c)
    {
        echo "Result: ", $a + $b;
    }
    else if ($a > $b && $c > $b)
    {
        echo "Result: ", $a + $c;
    }
    else
    {
        echo "Result: ", $b + $c;
    }

File: task5.cpp
#include <iostream>
using namespace std;
int main()
{
    float a, b, c;
    cout << "Enter 3 numbers: " << endl;
    cin >> a >> b >> c;
    if (a > c && b > c)
    {
        cout << "Result: " << a + b;
    }
    else if (a > b && c > b)
    {
        cout << "Result: " << a + c;
    }
    else
    {
        cout << "Result: " << b + c;
    }
}
